app/assets/javascripts/models/knockout_extensions.js.coffee

```diff
@@ -159,24 +159,6 @@ do (ko, $=jQuery) ->
           try
             $el.echoforms(form: form, prepopulate: options.prepopulatedFields())
 
-            # this needs to be done to trigger echoform validation otherwise these nested values will be pruned.
-            if shortName?
-              projectionOptions = $('#' + shortName + '-projectionSettings:visible')
```

Comment:
For example, this finds the projection options. If ```shortname``` is hard to find in the html, you can use wildcard to get all ```*-projectionSettings``` that are visible.